docker-compose.yml

```diff
@@ -39,6 +39,8 @@ fpm:
     - mailhog:mailhog
   volumes_from:
     - html
+  volumes:
+    - ./config/fpm/drushrc.php:/etc/drush/drushrc.php:ro
```

Comment:
Can you drop drushrc.php into config/drush and map that entire directory into /etc/drush instead of a single file, please?